### PR TITLE
Support "Copy Attributes to clipboard"

### PR DESCRIPTION
**Pull request by khanhdodang**

### CODE REVIEW

**Comment by khanhdodang** [comment]
Update screenshot for review
Screen Shot 2019-04-25 at 12 19 20 AM

**Comment by dpgraham** [comment]
Perfect... I was just about to ask for that

**Comment by dpgraham** [comment]
@jlipps what do you think?

File: app/renderer/components/Inspector/SelectedElement.js [hunk 1]
```diff
@@ -95,6 +96,18 @@ class SelectedElement extends Component {
...
+    const copyAttributesToClipboard = (dataSource) => {
+      let result = '';
+      Object.entries(dataSource).forEach(function (obj) {
+        if (obj[1].key === 'bounds') {
+          result += `${obj[1].key},"${obj[1].value}"\n`;
+        } else {
+          result += `${obj[1].key},${obj[1].value}\n`;
+        }
+      });
+      return result.toString();
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js]: toString is redundant

File: app/renderer/components/Inspector/SelectedElement.js [hunk 2]
```diff
@@ -95,6 +96,18 @@ class SelectedElement extends Component {
       });
     }
 
+    const copyAttributesToClipboard = (dataSource) => {
+      let result = '';
+      Object.entries(dataSource).forEach(function (obj) {
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js]: I'd rather reduce dataSource entries to a json object and then return its stringified representation

File: app/renderer/components/Inspector/SelectedElement.js [hunk 3]
```diff
@@ -126,6 +139,12 @@ class SelectedElement extends Component {
             >
               {t('Clear')}
             </Button>
+            <Tooltip title={t('Copy Attributes to Clipboard')}>
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js]: is this text present in resources?

File: app/renderer/components/Inspector/SelectedElement.js (hunk 2 above)
Comment (mykola-mokhnach) [reply app/renderer/components/Inspector/SelectedElement.js]: also, replacing `obj` with `[, smth]` would be more readable

Comment (mykola-mokhnach) [reply app/renderer/components/Inspector/SelectedElement.js]: or even `[,{key, value}]`

File: app/renderer/components/Inspector/SelectedElement.js (hunk 3 above)
Comment (dpgraham) [reply app/renderer/components/Inspector/SelectedElement.js]: @khanhdodang you'll need to add this to `assets/locales/en/translation.json`

File: app/renderer/components/Inspector/SelectedElement.js (hunk 2 above)
Comment (khanhdodang) [reply app/renderer/components/Inspector/SelectedElement.js]: Hi @mykola-mokhnach 

The current Array looks like

```
[{"key":"elementId","value":"ebf5e6eb-8b9e-409e-adbd-1c6030d461e4","name":"elementId"},{"key":"index","value":"0","name":"index"},{"key":"text","value":"","name":"text"},{"key":"class","value":"android.view.ViewGroup","name":"class"},{"key":"package","value":"com.dozuki.ifixit","name":"package"},{"key":"content-desc","value":"","name":"content-desc"},{"key":"checkable","value":"false","name":"checkable"},{"key":"checked","value":"false","name":"checked"},{"key":"clickable","value":"false","name":"clickable"},{"key":"enabled","value":"true","name":"enabled"},{"key":"focusable","value":"false","name":"focusable"},{"key":"focused","value":"false","name":"focused"},{"key":"scrollable","value":"false","name":"scrollable"},{"key":"long-clickable","value":"false","name":"long-clickable"},{"key":"password","value":"false","name":"password"},{"key":"selected","value":"false","name":"selected"},{"key":"bounds","value":"[0,0][1080,1920]","name":"bounds"},{"key":"resource-id","value":"android:id/decor_content_parent","name":"resource-id"},{"key":"instance","value":"0","name":"instance"}]
```
and I will keep the event of the button likes

```
 <Tooltip title={t('Copy Attributes to Clipboard')}>
              <Button
                disabled={!elementId}
                id='btnCopyAttributes' icon="copy"
                onClick={() => clipboard.writeText(JSON.stringify(dataSource))}/>
            </Tooltip>
```

Does it make sense?

File: app/renderer/components/Inspector/SelectedElement.js (hunk 3 above)
Comment (khanhdodang) [reply app/renderer/components/Inspector/SelectedElement.js]: Hi @dpgraham , I addressed your comment.

**Comment by khanhdodang** [comment]
Hi @mykola-mokhnach any updates?

**Pull request merged**